src/core/cheerio.ts

```diff
@@ -13,29 +14,36 @@ export interface Release {
  * @param html - Plain html text.
  * @returns cheerio object.
  */
-export function loadHTML(html: string): cheerio.CheerioAPI {
-  const $ = cheerio.load(html);
+export function loadHTML(html: string): CheerioAPI {
+  const $ = load(html);
   return $;
 }
 
 /**
- * Returns an array of objects with releases.
+ * Returns an array of objects.
  * @param c - cheerio object.
- * @param isNewRelease - boolean, if it's true return `new releases` otherwise `coming soon releases`.
+ * @param isNewRelease - boolean indicating whether to scrape new releases (`true`) or coming soon releases (`false`).
+ * @returns an array of objects representing the scraped releases.
  */
-export function getReleases(c: cheerio.CheerioAPI, isNewRelease: boolean): Release[] {
+export function getReleases(c: CheerioAPI, isNewRelease: boolean): Release[] {
   const releases: Release[] = [];
   let classname: string;
 
   isNewRelease ? (classname = NRCN) : (classname = CSCN);
 
-  const scrapReleases = c(classname);
-
-  scrapReleases.each((_i, release) => {
-    const artist = c(release).find(ARTISTCN).text();
-    const title = c(release).find(TITLECN).text();
-    const label = c(release).find(LABELCN).text();
-    const cover = c(release).find(COVERCN).attr('data-src');
+  const nodes = c(classname);
+
+  nodes.each((_i, node) => {
+    const artist = c(node)
+      .find(ARTISTCN)
+      .text()
+      .replace(/\s{2,}/g, ' ')
+      .replace(/\n/g, '')
+      .replace(/\s\/\s/g, ' / ')
+      .trim();
+    const title = c(node).find(TITLECN).text().replace(/\n/g, '').trim();
+    const label = c(node).find(LABELCN).text().replace(/\n/g, '').trim();
+    const cover = c(node).find(COVERCN).attr('data-src')?.replace(/\n/g, '').trim();
```

Comment:
In the artist value, it is necessary to remove extra spaces and new lines because there are some inconsistencies.
i'm 100% that in cover it's not required the regex and the trim, but not sure on title and labe, that's why i only left to remove new lines.